src/runtime/core/CMakeLists.txt

```diff
@@ -120,8 +120,10 @@ target_link_libraries(
   -L${ROCM_LIBRARIES_DIR}
   -L${CMAKE_INSTALL_PREFIX}/lib -lamd_hostcall -Wl,-rpath,${CMAKE_INSTALL_PREFIX}/lib
   -Wl,--enable-new-dtags
-  -Wl,-rpath,\$ORIGIN/../hsa/lib
-  -Wl,-rpath,\$ORIGIN/../lib
+  -Wl,-rpath,\$ORIGIN/../../hsa/lib
+  -Wl,-rpath,\$ORIGIN/../../lib
+  -Wl,-rpath,${ROCM_LIBRARIES_DIR}/../hsa/lib
+  -Wl,-rpath,${ROCM_LIBRARIES_DIR}
```

Comment:
Same comment as above. The runtime location of the dependent libraries for aomp should just $ORIGIN since all of its libraries are colocated under aomp/lib directory. If that will be case, please add \$ORIGIN as the first entry in the rpath.

Also, @ronlieb please ask devops folks to try the patch in a ROCm build as that is the official build, not the same as my workspace build of ROCm.